pennylane/devices/default_tensor.py

```diff
@@ -147,6 +147,10 @@ class DefaultTensor(Device):
     """A PennyLane device to perform tensor network operations on a quantum circuit using
     `quimb <https://github.com/jcmgray/quimb/>`_.
 
+    This device is designed to simulate large-scale quantum circuits using tensor networks. For small circuits, other devices like ``default.qubit`` are more suitable.
+    Different methods can be used to simulate the quantum circuit, and the backend uses the ``quimb`` library to perform the tensor network operations.
+    Currently, only the Matrix Product State (MPS) method is supported, based on the ``quimb``'s ``CircuitMPS`` class.
+
     Args:
         wires (int, Iterable[Number, str]): Number of wires present on the device, or iterable that
             contains unique labels for the wires as numbers (i.e., ``[-1, 0, 2]``) or strings
```

Comment:
We could omit the `shots` argument for now. Would there be any challenges to do this?

Reply:
I agree, we can safely remove it for this quarter since we are computing expectations value and variances analytically. Should we remove it?

Reply:
Yes please